services/QuillLMS/client/app/bundles/Teacher/components/accounts/new/select_user_type.jsx

```diff
@@ -1,17 +1,19 @@
 import * as React from 'react'
-import { Card } from '../../../../Shared/index'
+import { Card, networkIcon, STUDENT, TEACHER, INDIVIDUAL_CONTRIBUTOR } from '../../../../Shared/index'
 
 import AssignActivityPackBanner from '../assignActivityPackBanner'
 import { requestPost, } from '../../../../../modules/request/index'
 
-const studentPencilImg = `${process.env.CDN_URL}/images/onboarding/student-pencil.svg`
-const teacherChalkboardImg = `${process.env.CDN_URL}/images/onboarding/teacher-chalkboard.svg`
+const studentPencilImg = `${process.env.CDN_URL}/images/onboarding/student-pencil-colored.svg`
+const teacherChalkboardImg = `${process.env.CDN_URL}/images/onboarding/teacher-chalkboard-colored.svg`
+const homeSchoolImg = `${process.env.CDN_URL}/images/onboarding/home-building-colored.svg`
 
-class SelectUserType extends React.Component {
+export class SelectUserType extends React.Component {
   setRoleOnSession = (role) => {
+    const urlParam = role === INDIVIDUAL_CONTRIBUTOR ? TEACHER : role
     requestPost(`${process.env.DEFAULT_URL}/account/role`, { role, },
```

Comment:
I would take `{ role, }` out of this line here because you aren't using it now and it allows for ambiguity with the function param.

Reply:
Nevermind! Misread that!